composer/trainer/trainer.py

```diff
@@ -1853,7 +1840,8 @@ def _train_batch(self, use_grad_scaling: bool):
                     # Skip return and rerun after handling oom
                     continue
             # Log grad_accum and return loss if we've completed without OOMing.
-            self.logger.data_batch({'trainer/grad_accum': self.state.grad_accum})
+            self.logger.log_metrics({'trainer/grad_accum': self.state.grad_accum},
```

Comment:
does the `logger` have access to state? If so, we can make `step` default to the timestamp to make the code more concise?

Reply:
Already defaults to that. Just forgot to remove the optional argument when calling the function